prometheus_client/__init__.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
This file does of a main in it. It doesn't do much but I think this belongs here still.